rosidl_generator_c/package.xml

```diff
@@ -9,6 +9,7 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
 
+  <build_depend>ament_cmake_ros</build_depend>
```

Comment:
This needs to be a `buildtool_depend` dependency.

With the above comment the `buildtool_depend` entry for `ament_cmake` can be removed.